Unix/http/http.c

```diff
@@ -750,7 +766,7 @@ static void _ResetWriteState(
     socketData->httpErrorCode = 0;
 //    socketData->isAuthorised   = FALSE;
     socketData->authFailed     = FALSE;
-    socketData->encryptedTransaction = FALSE;
+//    socketData->encryptedTransaction = FALSE;
```

Comment:
Prefer that this be removed rather than commented out if not necessary. Same true two lines above as well (socketData->isAuthorized).